cmake/PythonInstallCLITools.py

```diff
@@ -58,7 +58,7 @@
 from sys import argv, executable
 
 # expecting one command line argument - the path to the python_lib folder to place the pip package
-pkgs = {'energyplus_launch': '3.7.4', 'energyplus_transition_tools': '2.1.4'}
+pkgs = {'energyplus_launch': '3.7.4', 'energyplus_transition_tools': '2.1.4', 'ghedesigner': '1.5'}
```

Comment:
This will be more meaningful in the next PR. I could remove it from here if anyone thinks it is necessary.